src/app/pages/elements/form/templates/VerticalDynamicForm.html

```diff
@@ -1,6 +1,10 @@
 <button theme="secondary" (click)="nomyform.showAllFields()">Show All Fields</button>
 <button theme="secondary" (click)="nomyform.showOnlyRequired()">Show Required Fields</button>
-<novo-dynamic-form layout="vertical" class="dynamic" [controls]="dynamicVertical" [(form)]="dynamicVerticalForm" #nomyform></novo-dynamic-form>
+<novo-dynamic-form layout="vertical" class="dynamic" [controls]="dynamicVertical" [(form)]="dynamicVerticalForm" #nomyform>
+  <ng-template novoTemplate="custom-demo-component" type="custom-demo-component" let-control let-form="form">
```

Comment:
What does the `type` do? That is already passed in the `novoTemplate` right? The value is the type.

Reply:
type has been removed now